WARNING: No test specified, defaulting to first test defined (multdyn_deepstate_test_dlmLplCpp_test)
TRACE: Initialized test input buffer with data from `/home/akhila/fuzzer_packages/fuzzedpackages/multdyn/inst/testfiles/dlmLplCpp/dlmLplCpp_output/12705850751e893b036929a777a1e4cf63fbc5b3.fail`
TRACE: Running: multdyn_deepstate_test_dlmLplCpp_test from /home/akhila/fuzzer_packages/fuzzedpackages/multdyn/inst/testfiles/dlmLplCpp/dlmLplCpp_DeepState_TestHarness.cpp(13)
input starts
Missing value
EXTERNAL: qs v0.23.4.

Yt_ values: 1.27891e-188 7.52749e+12 -2.14488e+105 inf -2.02048e-44 nan 3.42032e-93 8.27932e-269 -4.36409e-38 inf 0
Missing value
Ft_ values:     nan 4.59987e+18 6.61249e-257    -inf
    inf 1.51667e-225     nan 1.01363e+74

delta values: 1.49439e+214
m0_ values: -7261.59
CS0_ values: 1.80254e+45
n0 values: -1.19985e-167
d0 values: -1.25433e+73
input ends

error: Mat::col(): index out of bounds
terminate called after throwing an instance of 'std::logic_error'
  what():  Mat::col(): index out of bounds
ERROR: Failed: multdyn_deepstate_test_dlmLplCpp_test
ERROR: Test case /home/akhila/fuzzer_packages/fuzzedpackages/multdyn/inst/testfiles/dlmLplCpp/dlmLplCpp_output/12705850751e893b036929a777a1e4cf63fbc5b3.fail failed
